### ISSUE TITLE
twisted.python.zipstream needs to be ported to Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8607]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8607|
|Type|enhancement|
|Created|2016-07-16 10:29:24Z|
|Branch|https://github.com/twisted/twisted/tree/8607-rodrigc-zipfile|

twisted.python.zipstream does not work on Python 3

<details><summary>Searchable metadata</summary>

```
trac-id__8607 8607
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8607_rodrigc_zipfile 8607-rodrigc-zipfile
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1468664964545513 1468664964545513
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_2"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/339

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset 2049661a6530fa03c34dead92c3dbbb5cacb5c89
```
#!CommitTicketReference repository="" revision="2049661a6530fa03c34dead92c3dbbb5cacb5c89"
Merge pull request #339 from twisted/8607-rodrigc-zipfile

Author: rodrigc
Reviewer: acabhishek942
Fixes: #8607

Port zipstream to Python 3
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
